pt-BR translation minor improvements/translation

vicsjoo opened the pull request:
Adding to the Brazilian Portuguese translation of ReactOS.

binarymaster reviewed (changes requested):
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Also it's recommended to set up your GitHub account correctly, so maintainers can use Squash and Merge button without getting the author field modified back to nickname by GitHub, it's done in two steps:

1. Set your full name in [GitHub profile settings](https://github.com/settings/profile)
1. Uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails)

Regards,

vicsjoo commented:
@binarymaster, thank you for replying. I added my name and e-mail to the repo settings on GitHub Desktop, ammended the commit and force pushed it to my branch, however github also changed my e-mail to noreply@github.com. I think it is because I first edited the translation on Github Web editor. If it is still not as the guidelines require, you can close this PR and I can edit it on VS2019, which should add my name and e-mail correctly.
